src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -144,6 +152,12 @@ CHIP_ERROR writeAdminsIntoFabricsListAttribute()
         err = CHIP_ERROR_PERSISTED_STORAGE_FAILED;
```

Comment:
Not in this diff, but the call to `writeFabricAttribute` with one arg is broken on big-endian, afaict...

This needs to be done very carefully.  Ideally we would have better tests....

Reply:
Will file an issue to fix this + add tests

Reply:
https://github.com/project-chip/connectedhomeip/issues/8210